monai/data/meta_tensor.py

```diff
@@ -510,9 +510,16 @@ def new_empty(self, size, dtype=None, device=None, requires_grad=False):
             self.as_tensor().new_empty(size=size, dtype=dtype, device=device, requires_grad=requires_grad)
         )
 
-    def clone(self):
-        """returns a copy of the MetaTensor instance."""
-        new_inst = MetaTensor(self.as_tensor().clone())
+    def clone(self, *_args, **kwargs):
```

Comment:
May I know why add this `*_args`? Maybe just `def clone(self, **kwargs)`?

Thanks.